test/razorpay/test_payment.rb

```diff
@@ -79,5 +142,35 @@ def test_payment_capture_without_fetch
       payment = Razorpay::Payment.capture(@payment_id, amount: 5100)
       assert_equal 'captured', payment.status
     end
+
+    def test_payment_edit
+
+      payment_attr = {
+        "notes": {
+          "key1": 'value1',
+          "key2": 'value2'
+        }
+      } 
+      
+      stub_patch(%r{payments\/#{"pay_IY4ljejpy9KUr9"}$}, 'fake_update_payment', payment_attr.to_json)
+      payment = Razorpay::Payment.edit("pay_IY4ljejpy9KUr9",payment_attr.to_json)
+      assert_equal 'captured', payment.status
```

Comment:
wrong assertion, api only allows editing notes - https://razorpay.com/docs/api/payments/#update-the-payment

Reply:
Done